### PR TITLE
Use ExceptT correctly instead of throwIO.

### PR DESCRIPTION
**Pull request by ksaric**
Use ExceptT correctly instead of throwIO.
Last `throwIO` I found.

### CODE REVIEW

File: cardano-node/src/Cardano/Common/LocalSocket.hs [hunk 1]
```diff
@@ -8,14 +9,26 @@ module Cardano.Common.LocalSocket
   )
 where
 
+import           Prelude (show)
 import           Cardano.Prelude
 
+import           Control.Monad.Trans.Except.Extra (firstExceptT, hoistEither)
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Common/LocalSocket.hs]: Should be:
```
import           Control.Monad.Trans.Except.Extra (firstExceptT, hoistEither)
import           System.Directory (createDirectoryIfMissing, removeFile)
```

**Comment by erikd** [comment]
@ksaric This need to be rebase against master and force pushed.

**Pull request merged**